content/blog/my-first-day-at-skyworkz.md

```diff
@@ -0,0 +1,52 @@
+---
+id: blog
+title: My first day at Skyworkz
```

Comment:
Only a part of your blog is about this? 😉 For a blogpost with this title I would expect a post where you tell us more about who you are and what you do, etc. But this blogpost was more about going to a conference on your first day with a segway to Blipz. Which is fine, but I think the title should reflect that. 

How about: "Joining Skyworkz and hitting a conference on my first day"